.github/actions/build/action.yml.j2

```diff
@@ -0,0 +1,94 @@
+name: build
+
+description: Build specified project
+
+inputs:
+  build-type:
+    required: true
+    type: string
+    description: One of ci / release
+  target-device:
+    required: true
+    type: string
+  platform:
```

Comment:
is this host_platform or target_platform?

Reply:
Host platform. Updated.